### DISCUSSION TITLE
failure email. confusing bug in community/community

### DISCUSSION THREAD

**Discussion by jmerc77** [community/community#59045] [Discussions]
### Select Topic Area

Bug

### Body

got this email after posting in general on community:

> [community/community] On Create: Remove Governed Label workflow run | [community/community] On Create: Remove Governed Label workflow run |  
> 
> [community/community] On Create: Remove Governed Label workflow run
> 
> [community/community] On Create: Remove Governed Label workflow run
>  
> On Create: Remove Governed Label: All jobs have failed
>  
> [View workflow run](https://github.com/community/community/actions/runs/5363103640)
>  
> Check labels	
> On Create: Remove Governed Label / Check labels
> Failed in 6 seconds

received about 24h after post. confused as to why. i believe it is a bug. there seem to be recent updates to the repo in the file containing the Remove Governed Label check. maybe one broke something. the post exists so maybe it can be ignored but is quite annoying. also, there are multiple of these failures in the logs from multiple users. currently over 400.

edit: i moved the message in question to here from general because it is a feature request.

**Comment by jshtuazon2** [comment]
Application ID: 51984344
Referrence# 230716-017879
